app/models/tasks/copy_media_task.rb

```diff
@@ -48,18 +48,29 @@ def update_media_resource
         media_resource.duration = porter_callback_transcode[:Duration]&.to_f&./ 1000
       end
 
-      # change status, for invalid and bad-duration files
-      if media_resource.invalid?
-        media_resource.status = "invalid"
-      elsif info[:Audio].try(:[], :DurationDiscrepancy).to_i > 500
-        media_resource.status = "processing"
-        Tasks::FixMediaTask.create! { |t| t.owner = media_resource }.start!
-      end
+      # change status, if metadata doesn't pass validations
+      media_resource.status = "invalid" if media_resource.invalid?
+    end
+
+    media_resource.save!
+
+    if media_resource.status_complete? && (bad_audio_duration? || bad_audio_bytes?)
+      fix_media!
+    else
+      slice_media!
```

Comment:
Slight logic change here - if we need to fix the file, _don't_ slice it.  Wait until after we fix it (added this same slice_media logic to the fix_media_task).